phpseclib/Crypt/RSA/Formats/Keys/PSS.php

```diff
@@ -151,8 +151,6 @@ public static function load($key, $password = '')
         $result['MGFHash'] = str_replace('id-', '', $params['maskGenAlgorithm']['parameters']['algorithm']);
         if (isset($params['saltLength'])) {
```

Comment:
We can remove the isset check as well. saltLength is always going to be there, even if the orig key didn't explicitly specify it.

Thanks!

Reply:
Nope, because $params is an empty array if there are no parameters.